Solutions/Cisco ETD/Data Connectors/azuredeploy_CiscoETD_API_FunctionApp.json

```diff
@@ -0,0 +1,324 @@
+              "version": "1.0",
+              "sourceId": "[variables('workbookSourceId')]",
+              "category": "[variables('workbookType')]"
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2018-11-01",
+            "name": "[variables('FunctionName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', variables('FunctionName'))]",
+                "[resourceId('Microsoft.Insights/components', variables('FunctionName'))]"
+            ],
+            "kind": "functionapp,linux",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "name": "[variables('FunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true,
+                "siteConfig": {
+                    "linuxFxVersion": "python|3.8"
+                }
+            },
+            "resources": [
+                {
+                    "apiVersion": "2018-11-01",
+                    "type": "config",
+                    "name": "appsettings",
+                    "dependsOn": [
+                        "[concat('Microsoft.Web/sites/', variables('FunctionName'))]"
+                    ],
+                    "properties": {
+                        "FUNCTIONS_EXTENSION_VERSION": "~4",
+                        "FUNCTIONS_WORKER_RUNTIME": "python",
+                        "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]",
+                        "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]",
+                        "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('FunctionName'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('FunctionName')), '2015-05-01-preview').key1)]",
+                        "WorkspaceID": "[parameters('WorkspaceID')]",
+                        "SharedKey": "[parameters('SharedKey')]",
+                        "ApiKey": "[parameters('ApiKey')]",
+                        "ClientId": "[parameters('ClientId')]",
+                        "ClientSecret": "[parameters('ClientSecret')]",
+                        "Region": "[variables('ETDregion')[parameters('Region')]]",
+                        "Verdicts": "[parameters('Verdicts')]",
+                        "WEBSITE_RUN_FROM_ZIP": "https://ciscoetd.blob.core.windows.net/etd/CiscoETDAzureFunction.zip"
```

Comment:
Hello @sepintocisco, Link mentioned over here for zip file does not redirects to AZure repo.        
Can you explain me about the link mentioned by you

Reply:
Hi, it is the same zip file present in the PR, that is hosted elsewhere. It needs to be changed to the github url that directs to the master branch of the repository, thus have kept it on hold. The same needs to be done for the url the deploy button redirects to. If you could guide us with the link to master branch, it would be of great help.

Reply:
Can the "WEBSITE_RUN_FROM_PACKAGE" url be from the repositories zip file? like this url "https://github.com/Cisco-Email-Security/Azure-Sentinel/blob/CiscoETD/Solutions/Cisco%20ETD/Data%20Connectors/CiscoETDAzureFunction.zip" but it should point to the Azure/Azure-Sentinel master branch

Reply:
Hello @sepintocisco, We use shortlink (aka.ms) which redirects to the actual hyperlink of the repo. I'll update the link later, Dont worry